website/tutorial/managing-your-application-resources.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 6
+title: Managing your application resources
+description: Cover useful actions available to manage your application resources
+keywords: [podman desktop, podman, pod, Containerfile, Dockerfile]
+tags: [podman-desktop, managing-resources, managing-kubernetes-resources]
+---
+
+# Managing your application resources
+
+This tutorial covers the following tasks that you can perform to manage your application resources visually:
+
+- Managing containers and pods
+- Managing images
+- Managing volumes
+- Managing Kubernetes objects
+- Managing other resources
```

Comment:
Should these maybe be anchor links to the pages below? (Not sure if that is possible)

Reply:
we should avoid repeating words a lot when in bullet points, remove the "Managing" part

Reply:
@Firewall, we can link the bullet points to the relevant sections once they are published. I will keep a note of it and make this change.